### DISCUSSION TITLE
Release notes / upgrade tips

### DISCUSSION THREAD

**Discussion by martonb** [Mimetis/Dotmim.Sync#1235] [Ideas]
Hi,
in a pilot project we used Dotmim.Sync version 0.9.8. Today I had to do some tweaks in it and noticed there are new versions of this library, but found no details about the changes since 0.9.8, so I'm not sure if they are any breaking changes, or if I encounter interface changes, what is the recommended way of fixing them, etc. Thus I didn't upgrade.
So I want to suggest some small release notes for each released version.
Something similar to this: https://github.com/zzzprojects/EntityFramework-Plus/releases
Would help in the growth of this project ;)
Thanks for your efforts!

**Comment by Mimetis** [comment]
Thanks for the feedback
I know release notes are not consistent, but it requires a lot of time, and I'm alone to code & maintain this framework.

I try to make a release not if there is any backward compatibility with the previous version.
I think there is no breaking changes between **0.9.8** & **1.0.2** (unless I discover something :D )

**Reply by martonb** [reply]
Ok, that souns workable!
Meantime I attempted an upgrade, Dispose function was removed from AsyncState (iirc, not having the code in front of me), checked, it was doing nothing before, so not an issue.
Also some dependencies changed, will need config file redirects.
Hope this helps if anyone attempting the same...
Cheers!
